lib/importer/templates/review.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %} – {{ serviceName }} – GOV.UK Prototype Kit {% endblock %}
+{% block beforeContent %}
+    {{ govukBackLink({ text: "Back", href: "javascript:window.history.back()" }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Review options</h1>
+
+        <form action="." method="post">
+            <div class="govuk-button-group">
+                {{ govukButton({ text: "Submit" }) }}
+            </div>
+        </form>
```

Comment:
This page was intended to be reviewing validation issues rather than options...

Reply:
oops